packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -79,7 +100,10 @@ describe("Odsp Error", () => {
 			code: 400,
 		};
 		const networkError = errorObjectFromSocketError(socketError, "disconnect");
-		if (networkError.errorType !== DriverErrorType.genericNetworkError) {
+		if (
+			networkError.errorType !== OdspErrorTypes.genericNetworkError ||
+			!isIGenericNetworkError(networkError)
+		) {
```

Comment:
I think this should be enough, since the function already checks the `errorType`. If that's true (I didn't miss something), let's update the pattern in the other tests that we updated.

Also, can you add a note in the PR description regarding why we had to do this? I know we discussed it... and don't entirely recall the details, thus the ask 😅 .

```suggestion
		if (!isIGenericNetworkError(networkError)) {
```